python/tests/test_account.py

```diff
@@ -970,6 +970,30 @@ def test_send_and_receive_message_markseen(self, acfactory, lp):
         except queue.Empty:
             pass  # mark_seen_messages() has generated events before it returns
 
+    @pytest.mark.parametrize("mvbox_move", [True, False])
+    def test_markseen_message_and_mdn(self, acfactory, mvbox_move):
+        """Please only change this test if you are very sure that it will still catch the issues it catches now.
```

Comment:
This should probably be an ordinary comment, docstrings normally describe what the tests do.